library/cc/response_headers.h

```diff
@@ -5,16 +5,15 @@
 #include "headers.h"
 #include "response_headers_builder.h"
 
+class ResponseHeadersBuilder;
+
 class ResponseHeaders : public Headers {
 public:
+  ResponseHeaders(RawHeaders headers) : Headers(std::move(headers)) {}
```

Comment:
try to remember why i changed this from private + friend to public. choose one for all of the headers types